tsconfig.json

```diff
@@ -6,7 +6,9 @@
     "baseUrl": "./packages",
     "paths": {
       "@stoplight/elements-utils": ["./elements-utils/src"],
-      "@stoplight/elements": ["./elements/src"]
+      "@stoplight/elements": ["./elements/src"],
```

Comment:
Yeah, tried it, and it crashes builds.

Presumably because `@stoplight/elements` actually doesn't fall under `@stoplight/elements/*` rule, because it doesn't end with `/`.